packages/utilities/psammead-locales/moment/ps.js

```diff
@@ -24,9 +24,18 @@ var pashtoGregorianMonths = 'جنوري_فبروري_مارچ_اپریل_می_ج
 var jalaliFormats = ['D MMMM YYYY', 'LL'];
 
 moment.defineLocale('ps', {
-  // Gregorian Months
   months: pashtoGregorianMonths,
   monthsShort: pashtoGregorianMonths,
+
+  relativeTime: {
+    past: '%s مخکې', // %s 'ago'
+    s: 'یو څو ثانیې', // 'a few seconds'
+    m: 'یوه دقیقه', // 'a minute'
+    mm: '%d دقیقې', // %d 'minutes'
+    h: 'یو ساعت', // 'an hour'
+    hh: '%d ساعتونه', // %d 'hours'
+  },
+
```

Comment:
Think we should delete the blank line here and L29 for consistency with the other locales.